overlord/devicestate/handlers_install.go

```diff
@@ -589,6 +589,20 @@ func (m *DeviceManager) doRestartSystemToRunMode(t *state.Task, _ *tomb.Tomb) er
 		logger.Noticef("preseed data not present, will do normal seeding")
 	}
 
+	// if the model has a gadget snap, and said gadget snap has an install-device hook
+	// call systemctl daemon-reload to account for any potential side-effects of that
+	// install-device hook
+	hasHook, err := m.hasInstallDeviceHook(model)
+	if err != nil {
+		return fmt.Errorf("cannot obtain install device hook from model: %v", err)
```

Comment:
looking at the method I don't think we need to wrap the error, though other places using the method seem to wrap it in "internal error:" , which is a bit questionable, my suggestion is to drop the wrapping in all of them.

Reply:
Thanks, I removed the wrapping now